docker-compose.yaml

```diff
@@ -0,0 +1,41 @@
+version: "3"
+
+services:
+  todo_app:
+    build:
+      context: .
+      target: development
+    ports:
+      - 5000:80
+    volumes:
+      - type: bind
+        source: todo_app
+        target: /app/todo_app/todo_app
```

Comment:
This bind mount didn't allow hot-reloading for me (see comment above on the Dockerfile) but that's partly a Windows/WSL issue and partly the solution we've chosen to make testing work, so I'd argue not worth fixing here!